ts/features/pn/aar/screen/SendAARErrorScreen.tsx

```diff
@@ -1,12 +1,20 @@
+import { useEffect } from "react";
 import { useIOSelector } from "../../../../store/hooks";
 import { SendAARErrorComponent } from "../components/errors/SendAARErrorComponent";
 import { SendAARNotAddresseeComponent } from "../components/errors/SendAARNotAddresseeComponent";
 import { currentAARFlowStateType } from "../store/selectors";
 import { sendAARFlowStates } from "../utils/stateUtils";
+import { trackSendAARAccessDeniedScreenView } from "../analytics";
 
 export const SendAARErrorScreen = () => {
   const flowType = useIOSelector(currentAARFlowStateType);
 
+  useEffect(() => {
```

Comment:
why not use `useOnFirstRender`?

Reply:
I don't like the fact that it is a custom hook with an inside ref and that it allows the developer to not include the necessary dependencies in the callback. Moreover, in case of a (wrong) re-rendering, it does not run again but if the component is unmounted and mounted again (due to parent inclusion), then it does run again, against the fact that it is called "useOnFirstRender"